src/main/window-manager.ts

```diff
@@ -81,7 +85,8 @@ export class WindowManager extends Singleton {
         show: false,
         minWidth: 700,  // accommodate 800 x 600 display minimum
         minHeight: 500, // accommodate 800 x 600 display minimum
-        titleBarStyle: "hiddenInset",
+        titleBarStyle: isMac ? "hiddenInset" : "hidden",
+        frame: isMac,
```

Comment:
Does `frame: false` have any side-effects?

Reply:
I suggest it doesn't. https://www.electronjs.org/docs/latest/api/frameless-window The window is can be resized, minimized, maximized, closed and dragged.

Reply:
Anyway, I can't hide title bar in linux with any other methods provided.